nixos/modules/security/acme.nix

```diff
@@ -616,6 +617,8 @@ in {
   options = {
     security.acme = {
 
+      enableDebugLogs = mkEnableOption "debug logging in the acme script" // { default = true; };
```

Comment:
This works yeah? I'm always adding brackets in an abundance of caution

Reply:
Yeah I've been doing this for years now ;)